wdl-lint/src/util.rs

```diff
@@ -60,6 +71,17 @@ pub fn lines_with_offset(s: &str) -> impl Iterator<Item = (&str, usize, usize)>
     })
 }
 
+/// check if a program exists via `which`
```

Comment:
```suggestion
/// Check whether or not a program exists via `which`.
```

Note that you'll almost certainly have to do something different for Windows here.

Reply:
Yeah, can provide an alternate implementation via cfg